### ISSUE TITLE
Restrict hotkeys to files using tasks syntax

### ISSUE THREAD

**Issue by samuelmaddock** [sandy081/vscode-todotasks#12]
After installing this plugin, I was unable to use the `ctrl+enter` hotkey to insert a new line in any editor view. I think this plugin's hotkeys should be limited to files using the tasks syntax if possible.

This would also fix the issue in #10

**Comment by kidoman** [comment]
Had the same issue.

This:

```
                        {
                                "command": "task.new",
                                "key": "Ctrl+Enter",
                                "mac": "Cmd+Enter",
                                "when": "editorTextFocus"
                        }
```

essentially the base "Cmd+Enter" shortcut on VSCode as the "when" condition is exactly the same. Please add the language filter to the extensions shortcut.

**Comment by rbenson** [comment]
When I did #21, it fixed the keybindings to restrict  them to only work if the language was set to todo (Tasks in the status bar). I noticed this fix wasn't working anymore - I applied it locally since 21 isn't approved. Removing the language restriction allowed the bindings to work again, and not interfering with other languages. Can anybody confirm? @Sandy081 can you comment if something changed on the backend?

**Comment by kidoman** [comment]
Yep, it broke for me too... i had to disable this extension for now.

**Comment by rbenson** [comment]
So after I posted, something changed because it broke  again! I restarted VSC two times before posing

**Comment by sveggiani** [comment]
Broken for me too. VSC 1.17.1 here
